app/assets/javascripts/s3_aws_uploader/octavius-file-upload.js.erb

```diff
@@ -1,11 +1,25 @@
 "use strict";
 
-function octaviusUpload(fileInputSelector, progressBarSelector, inputReplacementSelector) {
+function OctaviusUpload(modelName, fieldName) {
+
```

Comment:
Next time think about what would happen if I input the params as cases not expected.